cognite_toolkit/cdf_tk/load.py

```diff
@@ -402,11 +401,12 @@ def fixup_resource(local: T_Resource, remote: T_Resource) -> T_Resource:
     def create_loader(
         cls,
         ToolGlobals: CDFToolConfig,
-        mode: Literal["load", "all_only"] = "load",
-        target_scopes: Literal["all", "resource_scoped_only", "all_scoped_only"] = "all",
+        target_scopes: Literal[
+            "all", "all_ignore_dataset", "all_scoped_ignore_dataset", "resource_scoped_only", "all_scoped_only"
```

Comment:
Any reason why we need `all_ignore_dataset` and `all_scoped_ignore_dataset`? These sounds exactly the same. 

In addition, is there a case where you would like to create all other scopes except for `dataset`? Isn't `all_scoped_only` enough?

Reply:
I read through you argument in slack once more, the above comment was a first impression, I suggest we have four options for target scope, `all`, `resource_scoped_only`, `all_scoped_only`, `skipped_validation`. The `skipped_validation` will then be used for the clean command. Wdyt?